### PR TITLE
Harsha release changes

### PR DESCRIPTION
**Pull request by HarshaMupparaju**
Code release Changes and validation done

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +22,6537 @@ runs/**/testing/test*/**/*.npy
 runs/**/testing/test*/**/*.npz
 tmp/
 .idea/
+data/databases/MipNeRF360/data/raw/unzipped_data/flowers.txt
```
Comment (NagabhushanSN95) [inline .gitignore]: Do not add individual images in gitignore. Just add their folder.

File: EnvironmentData/SimpleZipNeRF.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+conda create --name SimpleZipNeRF python=3.9 -y
```
Comment (NagabhushanSN95) [inline EnvironmentData/SimpleZipNeRF.sh]: Rename the file to Install_SimpleZipNeRF.sh

File: README.md [hunk 3]
```diff
@@ -12,7 +12,10 @@ Official code release accompanying the paper "Simple-RF: Regularizing Sparse Inp
 ### Python Environment
 Environment details are available in `EnvironmentData/SimpleZipNeRF.yml`. The environment can be created using conda
 ```shell
-conda env create -f SimpleZipNeRF.yml
+cd EnvironmentData
```
Comment (NagabhushanSN95) [inline README.md:15]: Shouldn't this be changed to bash file? There is no yml file now, right?

File: README.md [hunk 4]
```diff
@@ -27,8 +30,8 @@ Please follow the instructions in [database_utils/README.md](src/database_utils/
 The files `TrainerTester08_MipNeRF360.sh`, `TrainerTester04_NeRF_Synthetic.sh` contain the code for training, testing and quality assessment along with the configs for the respective databases.
 ```shell
 cd src/
-bash TrainerTester08_MipNeRF360.sh ../runs/training/train7140/Configs.gin --train_set_num 4
-bash TrainerTester04_NeRF_Synthetic.sh ../runs/training/train3012/Configs.gin --train_set_num 2
+bash TrainerTester08_MipNeRF360.sh ../runs/training/train7140/Configs.gin train_set_num=4
+bash TrainerTester04_NeRF_Synthetic.sh ../runs/training/train3012/Configs.gin train_set_num=2
 cd ../
 ```
```
Comment (NagabhushanSN95) [inline README.md:38]: Do not delete the gitkeep files if those folders are correct. Please add them back. If the folder path is incorrect, please add the gitkeep files in the corresponding folders.

File: runs/training/train7140/bonsai/ModelConfigs.json [hunk 5]
```diff
@@ -1,3 +1,3 @@
 {
-    "translation_scale": 0.21962338153662955
+    "translation_scale": 0.21962338153662947
```
Comment (NagabhushanSN95) [inline runs/training/train7140/bonsai/ModelConfigs.json:2]: Next time, don't push such small changes.

File: src/database_utils/nerf_synthetic/train_test_creators/TrainTestCreator01_UniformSampling.py [hunk 6]
```diff
@@ -1,124 +0,0 @@
-# Shree KRISHNAya Namaha
```
Comment (NagabhushanSN95) [inline src/database_utils/nerf_synthetic/train_test_creators/TrainTestCreator01_UniformSampling.py]: Why did you delete this file?

**Review by NagabhushanSN95 — CHANGES_REQUESTED** [review]
Next time, please try to create multiple pull requests with small changes instead of one PR with all changes.
1. It is easier to handle changes.
2. All bug fixes will be held up until everything is fixed.

File: src/database_utils/nerf_synthetic/train_test_creators/TrainTestCreator01_UniformSampling.py (hunk 6 above)
Comment (HarshaMupparaju) [reply src/database_utils/nerf_synthetic/train_test_creators/TrainTestCreator01_UniformSampling.py]: Added it back, vscode had too many Git changes so it was truncating.

File: README.md (hunk 4 above)
Comment (HarshaMupparaju) [reply README.md:38]: Added it back

**Comment by HarshaMupparaju** [comment]
I have a requirements.txt file in src and the bash script to create the environment in a different folder, is this fine? I did this because this was how it was for original ZipNeRF. Let me know if you want it to be changed.

File: README.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ Official code release accompanying the paper "Simple-RF: Regularizing Sparse Inp
 Environment details are available in `EnvironmentData/SimpleZipNeRF.yml`. The environment can be created using conda
 ```shell
 cd EnvironmentData
-conda env create -f SimpleZipNeRF_History.yml
+bash Install_SimpleZipNeRF.sh
 conda activate SimpleZipNeRF
```
Comment (NagabhushanSN95) [inline README.md]: Conda activate may not be needed after the bash script.

File: src/requirements.txt [hunk 8]
```diff
@@ -0,0 +1,23 @@
+numpy
```
Comment (NagabhushanSN95) [inline src/requirements.txt]: Move this to EnvironmentData

**Review by NagabhushanSN95 — APPROVED** [review]
There are a couple of minor changes. You can do them and merge.
